context/response/responses.go

```diff
@@ -0,0 +1,53 @@
+package response
+
+import (
+	beecontext "github.com/astaxie/beego/context"
+)
+
+// JSON renders value to the response as JSON
+func JSON(value interface{}, encoding ...bool) Renderer {
```

Comment:
This is mainly for code readability
The idea is that a controller method can be written like this:
```
func(c *TaskController) PutTaskById(id int64, task *models.Task) (*Status, error) {
	if /*not found */ {
		return nil, response.StatusCode(404)
	} else if /* needs redirect */ {
		return nil, response.Redirect("/somePath")
        } 

	return &Status{Status: "OK"}, nil
}
```
What do you think?